infra/core/monitor/applicationinsights.bicep

```diff
@@ -9,6 +13,16 @@ resource applicationInsights 'Microsoft.Insights/components@2020-02-02' = {
   kind: 'web'
   properties: {
     Application_Type: 'web'
+    WorkspaceResourceId: logAnalyticsWorkspaceId
+  }
+}
+
+module applicationInsightsDashboard 'applicationinsights-dashboard.bicep' =  if (includeDashboard) {
```

Comment:
I have an existing PR that adds a dashboard (and the related Bicep that's here), and my dashboard is pretty different than the azd one since I couldn't get browser and user stats to work. Do those stats actually work for all the azd samples? How?

Reply:
My PR: https://github.com/Azure-Samples/azure-search-openai-demo/pull/898

Reply:
It did work when we initially published, I haven't tested it in a while.  Would be good to see what diff you made and integrate into core if we can.

Reply:
I think my appinsights core changes are the same as yours (other than dashboard), so I'll keep in mine, we can test and merge it, then get the latest in yours.  Or the other way around. LMK what works for you.

Reply:
My dashboard is just vastly simplified to the stats that I could confirm were showing up, I don't think you'd want it in azd.

Reply:
Can you figure out a way to add yours without overwriting the one in core?

Reply:
I'm not sure that dashboards should be part of core? I would ideally customize our dashboard even more, to show openai-specific metrics, group by openai error rate, etc. It feels like something that should be passed in if that's possible.

And it's hard to have a "core" dashboard and add to it, since you literally have to configure the x/y position of each thing in the dashboard.

Reply:
Okay so how about, we bring in your default dashboard, but don't actually use it? (i.e. includeDashboard stays at false) I think I can refer to my simplified dash in main.bicep . And if someone wants to refer to the default one for ideas, they can still do so.

Reply:
yes, FYI, 
The default dashboard was scraped from the default one you get when you click new dashboard in the portal.